webpack.config.js

```diff
@@ -33,23 +33,31 @@ const WEBPACK_CONFIG = {
         } else {
```

Comment:
@jvigliotta https://github.com/akhenry/openmct-yamcs/pull/131 has the same changes, this file can be reverted.